bin/setup

```diff
@@ -0,0 +1,66 @@
+f this is a vagrant VM, cd to ~/opencraft on login
+if [[ $USER == 'vagrant' ]]; then
+    mkdir -p ~/opencraft
+    cd ~/opencraft
+    grep -Fq 'cd ~/opencraft' ~/.bashrc || echo 'cd ~/opencraft' >> ~/.bashrc
+fi
+
+# Need python-3.6 repository
+echo "Setting up python3.6 repository"
+sudo add-apt-repository ppa:jonathonf/python-3.6
```

Comment:
Let's use [deadsnakes](https://github.com/open-craft/ansible-accounting/pull/1/files#diff-ce1279346898431a34ccea1a7ade5060R9), since I'm using that in live environments through Ansible, and if we use Ansible to provision this in the future, it'll be nice to be consistent.